Dockerfile.ui-builder

```diff
@@ -0,0 +1,28 @@
+ARG image_tag=latest
+FROM elifesciences/pattern-library_assets:${image_tag} AS assets
+FROM elifesciences/php_cli:22434ef5bda09326d4c9347de7d8c2f1610a0b83
+
+USER elife
+ENV PROJECT_FOLDER=/srv/pattern-library-ui
+RUN mkdir ${PROJECT_FOLDER} && \
+    mkdir ${PROJECT_FOLDER}/public
+WORKDIR ${PROJECT_FOLDER}
+COPY --chown=elife:elife \
+    composer.json \
+    composer.lock \
+    ${PROJECT_FOLDER}/
+# customized command for composer
+RUN composer --no-interaction install
+
+COPY --chown=elife:elife \
+    core/styleguide ${PROJECT_FOLDER}/public/styleguide
+COPY --chown=elife:elife \
+    core/ ${PROJECT_FOLDER}/core
+COPY --chown=elife:elife config/ ${PROJECT_FOLDER}/config
+COPY --chown=elife:elife extras/ ${PROJECT_FOLDER}/extras
+COPY --chown=elife:elife bin/validate ${PROJECT_FOLDER}/bin/validate
```

Comment:
looked into putting the rest of `bin/` in `.dockerignore`, but it's counterintuitive